scripts/syncUNLCMSSites.php

```diff
@@ -109,3 +124,31 @@
         }
     }
 }
+
+
+//Clean up old unlcms development sites that no longer exist or have gone into production
+//ONLY if we have a list of base URLs from the call of unlcms. we wouldn't want to delete everything if that api call fails for some reason.
+if (!empty($cms_base_urls)) {
+    //Loop over all sites in the registry and remove the ones we need to
+    $cms_sites = new \SiteMaster\Core\Registry\Sites\All();
+    foreach ($cms_sites as $site) {
+        /**
+         * @var $site \SiteMaster\Core\Registry\Site
+         */
+        
+        //Only remove http://unlcms.unl.edu/ sites, as these are unlcms specific and development sites.
+        //They should not ever move to non-unlcms servers. Production sites might, so we can't assume anything for them.
+        if (0 !== strpos($site->base_url, 'http://unlcms.unl.edu/')) {
```

Comment:
Seems like it would be better to build a specific model to load just these sites to avoid the CPU and Memory overhead of loading everything from the DB.